tests/test_server.py

```diff
@@ -0,0 +1,155 @@
+        testNewQty = 1111
+        testUsedQty = 666
+        testOpenQty = 2018
+        testRestockLevel = 9
+        testRestockAmt = 4
+        data = json.dumps({PROD_ID: testProdId, PROD_NAME: testProdName,
+                    NEW_QTY: testNewQty, USED_QTY: testUsedQty, OPEN_BOXED_QTY: testOpenQty,
+                    RESTOCK_LEVEL: testRestockLevel, RESTOCK_AMT: testRestockAmt})
+        response = self.app.post(PATH_INVENTORY, data=data, content_type=JSON)
+
+        self.assertEqual(status.HTTP_201_CREATED, response.status_code)
+        self.assertTrue(response.headers.get(LOCATION, None) != None)
+        returnJson = json.loads(response.data)
+        self.assertIsNotNone(returnJson)
+        self.assertFieldsEqual(returnJson, testProdId, testProdName, testNewQty, testUsedQty,
+                testOpenQty, testRestockLevel, testRestockAmt)
+        response = self.app.get(PATH_INVENTORY)
+        data = json.loads(response.data)
+        self.assertEqual(status.HTTP_200_OK, response.status_code)
+        self.assertEqual(entryCount + 1, len(data))
+        self.assertIn(returnJson, data)
+                
+    def test_create_prod_info_withMandatoryFields(self):
+        entryCount = self.getEntryCount()
+
+        testProdId = 233
+        testProdName = 'bucai'
+        data = json.dumps({PROD_ID: testProdId, PROD_NAME: testProdName})
+        response = self.app.post(PATH_INVENTORY, data=data, content_type=JSON)
+
+        self.assertEqual(status.HTTP_201_CREATED, response.status_code)
+        self.assertTrue(response.headers.get(LOCATION, None) != None)
+        returnJson = json.loads(response.data)
+        self.assertIsNotNone(returnJson)
+        self.assertFieldsEqual(returnJson, testProdId, testProdName, DEFAULT_NEW_QTY,
+                DEFAULT_USED_QTY, DEFAULT_OPEN_BOXED_QTY, DEFAULT_RESTOCK_LEVEL,
+                DEFALUT_RESTOCK_AMT)
+        response = self.app.get(PATH_INVENTORY)
+        data = json.loads(response.data)
+        self.assertEqual(status.HTTP_200_OK, response.status_code)
+        self.assertEqual(entryCount + 1, len(data))
+        self.assertIn(returnJson, data)
+
+######################################################################
+# Utility functions
+######################################################################
+    def assertFieldsEqual(self, returnJson, expectProdId, expectProdName, expectNewQty,
+            expectUsedQty, expectOpenQty, expectRestockLevel, expectRestockAmt):
+        self.assertEqual(expectProdId, returnJson[PROD_ID])
+        self.assertEqual(expectProdName, returnJson[PROD_NAME])
+        self.assertEqual(expectNewQty, returnJson[NEW_QTY])
+        self.assertEqual(expectUsedQty, returnJson[USED_QTY])
+        self.assertEqual(expectOpenQty, returnJson[OPEN_BOXED_QTY])
+        self.assertEqual(expectRestockLevel, returnJson[RESTOCK_LEVEL])
+        self.assertEqual(expectRestockAmt, returnJson[RESTOCK_AMT])
+
+    def getEntryCount(self):
```

Comment:
Same naming style issue here. I'm not sure if we need to change the naming style for function assertFieldsEqual() (in line 137) too since it conforms with the naming style of the test suite... But I guess it might be better to at least unify the naming style of our own code.

Reply:
Addressed